docker/1.13/Dockerfile.gpu

```diff
@@ -1,70 +1,138 @@
+ && apt-get update \
+ && apt-get install -y --no-install-recommends \
+    libnvinfer5=${TF_TENSORRT_VERSION}-1+cuda10.0 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && rm /usr/lib/x86_64-linux-gnu/libnvinfer_plugin* \
+ && rm /usr/lib/x86_64-linux-gnu/libnvcaffe_parser* \
+ && rm /usr/lib/x86_64-linux-gnu/libnvparsers* \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz \
+ && tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION \
+ && ./configure && make && make install \
+ && apt-get update && apt-get install -y --no-install-recommends \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev libgdbm-dev \
+    libc6-dev libbz2-dev \
+ && rm -rf /var/lib/apt/lists/* \
+ && make && make install \
+ && rm -rf ../Python-$PYTHON_VERSION* \
+ && ln -s /usr/local/bin/pip3 /usr/bin/pip
+
+RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+
+# Some TF tools expect a "python" binary
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+
+RUN curl https://s3-us-west-2.amazonaws.com/tensorflow-aws/1.13/Serving/GPU/tensorflow_model_server --output tensorflow_model_server && \
+chmod 555 tensorflow_model_server && cp tensorflow_model_server /usr/bin/tensorflow_model_server && \
+rm -f tensorflow_model_server
 
 # nginx + njs
-RUN \
-    apt-get update && \
-    apt-get -y install --no-install-recommends curl gnupg2 && \
-    curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - && \
-    # find the correct linux distribution here - http://nginx.org/en/linux_packages.html
-    echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools && \
-    apt-get clean
-
-# cython, falcon, gunicorn, tensorflow-serving
-RUN \
-    pip3 install --no-cache-dir cython falcon gunicorn gevent requests grpcio protobuf && \
-    pip3 install --no-dependencies --no-cache-dir tensorflow-serving-api==1.13.0
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends curl gnupg2 \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends nginx nginx-module-njs \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
+
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install -U --no-cache-dir \
+    awscli==1.16.130 \
+    cython==0.29.10 \
+    falcon==2.0.0 \
+    gunicorn==19.9.0 \
+    gevent==1.4.0 \
+    requests==2.21.0 \
+    grpcio==1.24.1 \
+    protobuf==3.10.0 \
+# using --no-dependencies to avoid installing tensorflow binary
+ && ${PIP} install --no-dependencies --no-cache-dir \
+    tensorflow-serving-api==1.13.0
```

Comment:
I see a -gpu version:https://pypi.org/project/tensorflow-serving-api-gpu/. Can we look into what's different?

Reply:
Updated both 1.13 gpu and 1.14 gpu

Reply:
From [tensorflow release notes](https://github.com/tensorflow/serving/releases) 1.13.0 release notes

"New GPU version of TensorFlow Serving API PIP package. This depends on the tensorflow-gpu instead of tensorflow PIP package, but is otherwise identical. (commit: 525c1af)"

since we are not installing tensorflow its fine to use either